files/timelinejs/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "TimelineJS",
+  "repo": "NUKnightLab/TimelineJS",
+  "files": {
+    "basePath": "build/",
+    "include": ["css/**/*", "js/**/*"]
```

Comment:
You need to add `./` at the beginning of each folder to retain the directory format.